subsys/bluetooth/host/gatt.c

```diff
@@ -1318,6 +1318,24 @@ int bt_gatt_service_unregister(struct bt_gatt_service *svc)
 
 	return 0;
 }
+
+bool bt_gatt_service_is_register(struct bt_gatt_service *svc)
+{
+	bool ret = false;
```

Comment:
```suggestion
	bool registered = false;
```
Might as well give it a more descriptive name :)